### ISSUE TITLE
Allow to disable drive letter on home drive

### ISSUE THREAD

**Issue by dorianim** [linuxmuster/linuxmuster-linuxclient7#33]
Allow to disable drive letter on home drive. This should be done in a config file, and is therefore blocked by #32 
This also raises the question if there should also be an option to disable the drive letters in general, also for GPO-mounted drives. (overriding the option in Drives.xml)

**Comment by sedding** [comment]
Hej,

"plus one" for the option that turns off all drive letters at one go.

In Linux-environments the windows drive letters are irrelevant and even if one would like to have them, he would like have all of them... 
IMHO a option that turns off drive letter usage is completely sufficient - all or none - not at least for consistency reasons.

**Comment by dorianim** [comment]
Yes, I agree. But as I already mentioned, #32 has to be resolved first.

**Comment by jolly-jump** [comment]
Can you point me to where (H:) is hardcoded? I can't find it in the source of *this* repo. Maybe I just don't know my regexp. :shrug:  ?
Thanks

**Comment by dorianim** [comment]
https://github.com/linuxmuster/linuxmuster-linuxclient7/blob/a6115cfec0ea7fbfae13ec36f3e27b87c09934c9/usr/lib/python3/dist-packages/linuxmusterLinuxclient7/user.py#L151

H is not actually hardcoded. It is and ldap attribute.

**Comment by jolly-jump** [comment]
thanks. Yes, makes sense.
I changed that line for me to as a workaround

```
shareName = f"Home_auf_Server von {usernameString}"
```
